### PR TITLE
Add tests for Address component

### LINKED ISSUE #577: Add test for Address component
I think it's self explanatory, use cases:
- accounts not in the keyring
- eth account
- accounts with or without genesisHash
- external accounts
- with or without options...

### PR DESCRIPTION
**Pull request by Tbaut**
closes #577 

todo:

- [x] accounts with or without genesisHash (icon theme change and network label)
- [x] external accounts icon

### CODE REVIEW

File: packages/extension-ui/src/components/Address.tsx [hunk 1]
```diff
@@ -221,7 +221,7 @@ function Address ({ actions, address, children, className, genesisHash, isExtern
                   : undefined
               }
             >
-              {chain.name.replace('Relay Chain', '')}
+              {chain.name.replace(' Relay Chain', '')}
```
Comment (Tbaut) [inline packages/extension-ui/src/components/Address.tsx:224]: :tada: my test caught this tiny bug. Kusama and Polkadot labels actually had a white space at the front.

File: packages/extension-ui/src/components/Address.test.tsx [hunk 2]
```diff
@@ -13,10 +13,13 @@ import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { ThemeProvider } from 'styled-components';
 
+import allChains from '@polkadot/extension-chains/chains';
+import { Chain } from '@polkadot/extension-chains/types';
```
Comment (jacogr) [inline packages/extension-ui/src/components/Address.test.tsx:17]: Import type { ... }

Comment (Tbaut) [reply packages/extension-ui/src/components/Address.test.tsx:17]: Thanks, I'll do a cleanup soon, hope to finish this suite of tests today

**Comment by Tbaut** [comment]
Should be good to go now! I spotted what I think is a bug, the "Kusama" label used to be black (e.g here: https://github.com/polkadot-js/extension/pull/481), now it's orange. I'll check and fix this + add a test in another PR.

File: packages/extension-ui/src/components/Address.tsx (hunk 1 above)
Comment (jacogr) [reply packages/extension-ui/src/components/Address.tsx:224]: :)

**Comment by jacogr** [comment]
Mine is still black - start of the week.

**Review by jacogr — APPROVED** [review]
Brilliant!

**Comment by Tbaut** [comment]
Just had a look, it's expected since I didn't have the metadata imported on my testing instance. I'll use the mock metadata from Westend to add a test for that.

**Pull request merged**